dev/io.openliberty.io.smallrye.openapi.core.3.1/bnd.bnd

```diff
@@ -0,0 +1,39 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.io.smallrye.openapi.core.3.1; singleton:=true
```

Comment:
Are you sure that you want to include a minor version number in the project & bundle name?
```suggestion
Bundle-SymbolicName: io.openliberty.io.smallrye.openapi3.core; singleton:=true
```

Reply:
Hmm, maybe not? It's odd because we need 3.1. Version 3.0 will be for OpenAPI 3.0 with jakarta (we're using 2.x transformed)

Can't tell at the moment what their next version will be.

Reply:
Bundle names have been updated as suggested.